packages/block-library/src/pattern/edit.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
+import {
+	store as blockEditorStore,
+	InnerBlocks,
+} from '@wordpress/block-editor';
+
+const PatternEdit = ( { attributes, clientId, isSelected } ) => {
+	const selectedPattern = useSelect( ( select ) =>
+		select( blockEditorStore ).__experimentalGetParsedPattern(
+			attributes.slug
+		)
+	);
```

Comment:
Could we cache on slug?

```suggestion
	const selectedPattern = useSelect(
		( select ) =>
			select( blockEditorStore ).__experimentalGetParsedPattern(
				attributes.slug
			),
		[ attributes.slug ]
	);
```

Reply:
Still applicable.